Adding 1.10 App/Play store release content

hesterbruikman opened the pull request:
Edited overview of https://github.com/status-im/status-react/commits/release/1.10.x for App/Play store submission

### Review notes
Please suggest how to add https://github.com/status-im/status-react/commit/c33e2be76b87776fee5f17098db12fca597c1bb1
Not sure what the impact is on the user.

hesterbruikman committed:
Adding 1.10 App/Play store release content

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 0a185937 | [#1](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-11581/1/) | 2020-12-24 14:24:22 | ~14 min | `android-e2e` | [:package:`apk`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-201224-140950-0a1859-pr-x86.apk) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fstatus-im-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-201224-140950-0a1859-pr-x86.apk)|

hesterbruikman committed:
Removes Cancel and Speed up tx

As discussed, the feature is not yet reliable for public release. On occasion the nonce cannot be overwritten, resulting in 2 pending transactions.
